public/app/plugins/panel/timeseries/TimeSeriesTooltip.tsx

```diff
@@ -112,28 +110,14 @@ export const TimeSeriesTooltip = ({
       const v = seriesFrame.fields[i].values[dataIdxs[i]!];
       const display = field.display!(v); // super expensive :(
 
-      sortIdx.push(v);
       contentLabelValue.push({
         label: field.state?.displayName ?? field.name,
-        value: display ? formattedValueToString(display) : null,
+        value: display ? formattedValueToString(display) : '',
```

Comment:
I think we should display something here in case of failure. We have a similar portion in the table panel where there's a silent failure if there isn't a field display formatter. Silent failures are a nightmare to debug and cause significant user confusion when failures happen.